lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -203,6 +204,35 @@ class ReportDownloads
     grid = new Slick.Grid($table_placeholder, report_downloads_data, columns, options)
     grid.autosizeColumns()
 
+    $delete_btns = @$section.find(".delete-report")
+    $delete_btns.click (e) =>
+      table_row = jQuery(e.target.parentElement.parentElement)
+      filename_cell = table_row.find(".course-forums-data")
+      @$file_to_delete = filename_cell.text()
+      if confirm gettext 'Are you sure you want to delete the file ' + @$file_to_delete + '? This cannot be undone.'
```

Comment:
is the confirmation part of the spec? it seems like it might not be necessary to ask for confirmation because they can just regenerate the report if they accidentally delete it